### ISSUE TITLE
anyone have a working example with next js and this lib?

### ISSUE THREAD

**Issue by d3vhound** [mui-org/material-ui-pickers#1221]
Can't get this lib to work with next js and tried everything in each repo.

**Comment by dmtrKovalenko** [comment]
What the problems you have?

**Comment by d3vhound** [comment]
here's the stack trace I get with next.js by simply importing any component

```bash
SyntaxError: Unexpected token export
    at Module._compile (internal/modules/cjs/loader.js:760:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:827:10)
    at Module.load (internal/modules/cjs/loader.js:685:32)
    at Function.Module._load (internal/modules/cjs/loader.js:620:12)
    at Module.require (internal/modules/cjs/loader.js:723:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at Object.<anonymous> (/Users/devionvillegas/Developer/D3VHOUND/wh-admin/node_modules/@material-ui/pickers/dist/material-ui-pickers.js:7:32)
    at Module._compile (internal/modules/cjs/loader.js:816:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:827:10)
    at Module.load (internal/modules/cjs/loader.js:685:32)
    at Function.Module._load (internal/modules/cjs/loader.js:620:12)
    at Module.require (internal/modules/cjs/loader.js:723:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at Object.<anonymous> (/Users/devionvillegas/Developer/D3VHOUND/wh-admin/node_modules/material-table/dist/components/m-table-edit-field.js:32:16)
    at Module._compile (internal/modules/cjs/loader.js:816:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:827:10)
```

at first I thought it was `material-table` that was causing this but it's actually this line in material-table that throws:
```js
var _pickers = require("@material-ui/pickers");
```

**Comment by joncursi** [comment]
It's an issue with version `3.2.1` of this library. Downgrading to `3.2.0` works for me.

**Comment by dmtrKovalenko** [comment]
Hehe a lot of unexpected problems with 3.2.1

**Comment by MBach** [comment]
Same here: downgraded to 3.2.0

**Comment by stefanmihoc** [comment]
Is it just a matter of transpilation?

Cause it could be solved by an entry in `next.config.js` to include modules which needs to go through babel. See:

```javascript
module.exports = {
   transpileModules: [
     "@material-ui/pickers",
   ]
}
```

**Comment by dmtrKovalenko** [comment]
Fixed in v3.2.2
